src/api/css.spec.ts

```
@@ -133,6 +133,44 @@ describe('$(...)', () => {
+      it('should not track parentheses inside comments', () => {
+        const el = cheerio('<div></div>');
+        el.attr('style', 'color: red/*(*/; background: blue');
+        expect(el.css()).toStrictEqual({
+          color: 'red/*(*/',
+          background: 'blue',
+        });
+      });
+
+      it('should treat escaped characters as data', () => {
+        const el = cheerio('<div></div>');
+        el.attr('style', 'font-family: foo\\(bar; color: red');
```

ESLint

[error] `String.raw` should be used to avoid escaping `\`.

(unicorn/prefer-string-raw)